zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -526,13 +526,17 @@ private InterpreterContext getInterpreterContext() {
 
     Credentials credentials = note.getCredentials();
     if (subject != null) {
-      UserCredentials userCredentials =
-          credentials.getUserCredentials(subject.getUser());
+      UserCredentials userCredentials;
+      try {
+        userCredentials = credentials.getUserCredentials(subject.getUser());
+      } catch (IOException e) {
+        LOGGER.error("Unable to get Usercredentials. Working with empty UserCredentials", e);
```

Comment:
If we allow to fallback to empty UserCredentials, then I think we should use `LOGGER.warn`

Reply:
Changed.